### PR TITLE
Enhancement: better packaging scripts

### PR DESCRIPTION
**Pull request by floreal**
- build.sh: will build for release
- ./install.sh:
  - default $PREFIX set to /usr/local to avoid installing in /
  - use install over cp
- ./uninstall.sh:
  - remove installed files (leave directory structure though)
  - default $PREFIX set to /usr/local

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,7 +1,19 @@
-cargo build --release
-sudo cp target/release/mobydick $PREFIX/bin/xyz.gelez.mobydick
-sudo cp *.appdata.xml $PREFIX/share/appdata/
-sudo cp *.desktop $PREFIX/share/applications/
+#!/usr/bin/env sh
+if [ -z "$PREFIX" ]; then
+  PREFIX=/usr/local
+fi
+
+if [ -f target/release/mobydick ]; then
```
Comment (elegaanz) [inline install.sh]: Isn't it missing a `!`? It seems to me you are building only if there is already the file.

**Review by elegaanz — COMMENTED** [review]
Thank you for the PR! Just left a small comment

**Review by elegaanz — APPROVED** [review]
:ok_hand:

File: install.sh (hunk 1 above)
Comment (floreal) [reply install.sh]: Yes, it was missing.

**Pull request merged**